src/hooks/stations.ts

```diff
@@ -48,10 +48,27 @@ export const useBooking = () => {
         }),
     [BUCHUNGEN],
   );
+  const refreshBooking = useCallback(
+    async () => {
+      const oldBooking = await getCurrentBooking();
```

Comment:
If you use async await in one place, please use it in all.

Reply:
See 4e278367b526f5fa0e6482ded3b8d097ad70cf4f.